### PR TITLE
en: add doc for multi-cluster prometheus monitor

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added, or deleted? (Required)

Add docs about multi cluster prom monitoring.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1490
- Other reference link(s):

### CODE REVIEW

File: en/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    You can specify the store nodes by the static service discovery method. In the Thanos Query command line's starting parameters, add `--store=${cluster_name}-prometheus.${cluster_namespace}.svc.${cluster_domain}:10901` to specify the store node. Replace the variable with the actual value.
+
+    If you use other service discovery methods, refer to [thanos-service-discovery](https://thanos.io/tip/thanos/service-discovery.md).
+
+</div>
+<div label="Federation">
+
+### Using Prometheus Federation
+
+In the example of this section, the Federation Prometheus server is used as an entry point for unified storage and query. This method is recommended for small data volumes.
```
Comment (just1900) [inline en/deploy-tidb-monitor-across-multiple-kubernetes.md:198]:
```suggestion
In the example of this section, the Federation Prometheus server is used as an entry point for unified storage and query. This method is only considered for small data volumes.
```
we actually do not recommend using this.

File: en/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    ```shell
+    # set tidb version here
+    version=nightly
+    docker run --rm -i -v ${PWD}/dashboards:/dashboards/ pingcap/tidb-monitor-initializer:${version} && \
+    cd dashboards
+    ```
+
+    > **Note:**
+    >
+    > In the command above, `${version}` is the version of the Initializer image, which should be consistent with the TiDB version. However, currently only the `nightly` version of the Initializer image supports **multiple Kubernetes clusters** monitoring, so this example uses the `nightly` image.
```
Comment (just1900) [inline en/deploy-tidb-monitor-across-multiple-kubernetes.md:312]: since TiDB 6.0 has been released, maybe change the example to v6.0.0 here?

Comment (ran-huang) [reply en/deploy-tidb-monitor-across-multiple-kubernetes.md:312]: Sure. It's your call.

Comment (ran-huang) [reply en/deploy-tidb-monitor-across-multiple-kubernetes.md]: Updated in [5b7f056](https://github.com/pingcap/docs-tidb-operator/pull/1775/commits/5b7f0568c0c5ff373ae35cb15fd08b77843db841)

File: zh/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 3]
```diff
@@ -293,29 +295,35 @@ scrape_configs:
...
-    version=nightly
+    version=v6.0.0
     docker run --rm -i -v ${PWD}/dashboards:/dashboards/ pingcap/tidb-monitor-initializer:${version} && \
     cd dashboards
     ```
 
-    执行上述命令后，可以在当前目录下查看所有组件 dashboard 的 json 定义文件。
+    > **注意：**
+    >
+    > 上述命令中，`${version}` 为 Initializer 的镜像版本，应该和 TiDB 版本保持一致，但是目前仅 `v6.0.0` 版本的 Initializer 镜像适用于**多 Kubernetes 集群**监控，因此示例中使用 `v6.0.0` 版本的镜像。
```
Comment (qiancai) [inline zh/deploy-tidb-monitor-across-multiple-kubernetes.md:313]: v6.0.0 之后的版本是否计划也适用于**多 Kubernetes 集群**监控呢？如果是，或许可以加一个“及以上”避免以后还要回来改文档

File: en/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
+---
+title: Monitor a TiDB Cluster across Multiple Kubernetes Clusters
+summary:
```
Comment (ran-huang) [inline en/deploy-tidb-monitor-across-multiple-kubernetes.md:3]:
```suggestion
summary: Learn how to monitor a TiDB cluster across multiple Kubernetes clusters and aggregate the monitoring data into the common Prometheus multi-cluster monitoring systems.
```

File: zh/deploy-tidb-monitor-across-multiple-kubernetes.md (hunk 3 above)
Comment (just1900) [reply zh/deploy-tidb-monitor-across-multiple-kubernetes.md:313]: 适用

Comment (ran-huang) [inline zh/deploy-tidb-monitor-across-multiple-kubernetes.md:313]:
```suggestion
    > 上述命令中，`${version}` 为 Initializer 的镜像版本，应该和 TiDB 版本保持一致，但是目前仅 `v6.0.0` 版本及以上的 Initializer 镜像适用于**多 Kubernetes 集群**监控，因此示例中使用 `v6.0.0` 版本的镜像。
```

File: en/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+## Pull data from Prometheus
+
+The pull method pulls monitoring data from Prometheus instances in different Kubernetes clusters and aggregates the data into a global view. This section describes how to pull data [using Thanos Query](#using-thanos-query) and [using Prometheus Federation](#using-prometheus-federation).
+
+<SimpleTab>
+<div label="Thanos Query">
+
+### Using Thanos Query
+
+In the example of this section, one Thanos Sidecar is deployed for each Prometheus (`TidbMonitor`) component. The Thanos query component is used for aggregation queries. If you do not need long-term storage, you can skip the deployment of some components such as thanos-store and S3.
```
Comment (KanShiori) [inline en/deploy-tidb-monitor-across-multiple-kubernetes.md:105]:
```suggestion
In the example of this section, one Thanos Sidecar is deployed for each Prometheus (`TidbMonitor`) component. The thanos-query component is used for aggregation queries. If you do not need long-term storage, you can skip the deployment of some components such as thanos-store and S3.
```

**Review by KanShiori — APPROVED** [review]

**Review by KanShiori — APPROVED** [review]

File: en/deploy-tidb-monitor-across-multiple-kubernetes.md [hunk 6]
```diff
@@ -302,14 +302,14 @@ After collecting data using Prometheus, you can visualize multi-cluster monitori
...
-    version=v6.0.0
+    version=v6.1.0
     docker run --rm -i -v ${PWD}/dashboards:/dashboards/ pingcap/tidb-monitor-initializer:${version} && \
     cd dashboards
     ```
 
     > **Note:**
     >
-    > In the command above, `${version}` is the version of the Initializer image, which should be consistent with the TiDB version. However, currently only the `v6.0.0` version of the Initializer image supports **multiple Kubernetes clusters** monitoring, so this example uses the `v6.0.0` image.
+    > In the command above, `${version}` is the version of the Initializer image, which should be consistent with the TiDB version. Currently only `v6.0.0` and later versions of the Initializer image support **multiple Kubernetes clusters** monitoring.
```
Comment (qiancai) [inline en/deploy-tidb-monitor-across-multiple-kubernetes.md:312]:
```suggestion
    > In the command above, `${version}` is the version of the Initializer image, which should be consistent with the TiDB version. Currently, only `v6.0.0` and later versions of the Initializer image support **multiple Kubernetes clusters** monitoring.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**